chatkit-core/build.gradle

```diff
@@ -19,11 +19,13 @@ dependencies {
     }
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation "org.reduxkotlin:redux-kotlin:0.3.1"
     api 'com.squareup.okhttp3:okhttp:3.12.6'
     implementation 'com.google.code.gson:gson:2.8.5'
 
     testImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0'
     testImplementation 'com.google.truth:truth:0.40'
+    testImplementation 'com.willowtreeapps.assertk:assertk:0.21'
```

Comment:
as now it's preferred assertion library, it would be good to have it at the top and preceded the rest of dependencies with a comment explaining it's legacy

Reply:
👍 have documented using the because keyword as described [here](https://docs.gradle.org/current/userguide/declaring_dependencies.html#sec:documenting-dependencies)